test/helpers/kubectl.go

```diff
@@ -97,7 +98,8 @@ func (kub *Kubectl) GetPods(namespace string, filter string) *CmdRes {
 func (kub *Kubectl) GetPodNames(namespace string, label string) ([]string, error) {
 	stdout := new(bytes.Buffer)
 	filter := "-o jsonpath='{.items[*].metadata.name}'"
-	exit := kub.Execute(
+
+	exit, _ := kub.Execute(
```

Comment:
same here

Reply:
In this case, we only need if the exit was success or not. The error message is not essential.(A lot of them are supposed to fail)